### ISSUE TITLE
Undefined index: shortcode_values

### ISSUE THREAD

**Issue by sozonovalexey** [linkonoid/oc-shortcodesengine#1]
```
Undefined index: shortcode_values
/plugins/linkonoid/shortcodesengine/classes/ShortcodesManager.php line 135
```

Нужно добавить проверку на наличие параметров

**Comment by linkonoid** [comment]
Да, недочёт вижу – по коду при пустом shortcode_values repeater должна вываливаться ошибка, обязательно поправлю, точнее уже поправил (134 строка):

if (!empty($shortcode['shortcode_values'])) foreach ($shortcode['shortcode_values'] as $key => $value){

 

исправления выложу чуть позже, спасибо за информацию!
